### PR TITLE
Philadelphia

### PR DESCRIPTION
**Pull request by emilyhuntsman**
I just changed "D.C." to "DC" so it would fit in with the other states and entered a template for philly.

### CODE REVIEW

File: _emails/philadelphia.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+- Brian.Oneill@phila.gov
+- Mark.squilla@phila.gov
+- brian.abernathy@phila.gov
+- james.kenney@phila.gov
+- bobby@bobbyhenon.com
+subject: Philadelphia demands a People's Budget
+body: |
+ To whom it may concern: 
+
+My name is [insert name], and I am a resident of the [district number] district. I am emailing to demand the restructuring of the Philadelphia city budget in a way that prioritizes social services for communities and drastically minimizes spending on police.
```
Comment (teddywilson) [inline _emails/philadelphia.md:31]: L31-L38 should be indented

File: _emails/philadelphia.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+- Allan.Domb@Phila.gov
+- Cherelle.Parker@Phila.gov
+- Isaiah.Thomas@phila.gov
+- Darrell.Clarke@phila.gov
+- David.Oh@Phila.gov
+- Brian.Oneill@phila.gov
+- Mark.squilla@phila.gov
+- brian.abernathy@phila.gov
+- james.kenney@phila.gov
+- bobby@bobbyhenon.com
```
Comment (samuel-obrien25) [inline _emails/philadelphia.md:26]: bobbyhenon.com is suspended and likely not receiving emails. Can be updated to phila.gov email -- bobby.henon@phila.gov

File: _emails/philadelphia.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+subject: Philadelphia demands a People's Budget
+body: |
+ To whom it may concern: 
+
+ My name is [insert name], and I am a resident of the [district number] district. I am emailing to demand the restructuring of the Philadelphia city budget in a way that prioritizes social services for communities and drastically minimizes spending on police.
```
Comment (teddywilson) [inline _emails/philadelphia.md:31]: let's use all caps for the fields inside brackets.

[NAME], [DISTRICT NUMBER], etc.

File: _emails/philadelphia.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+ We are in the midst of widespread upheaval over the systemic violence of policing, embodied by the PPD's well documented history of disproportionately enacting violence against Black communities. We will no longer accept empty gestures and suggestions of “reform.” We are demanding that our voices be heard now, and that real change be made to the way this city allocates its resources.
+
+ The proposed $14 million budget increase for PPD is unacceptable and comes at the cost of affordable housing, support for essential city workers, and other transformative initiatives. These programs are needed now more than ever to alleviate the harms of the COVID-19 pandemic, especially for Black and Brown communities. I ask that the mayor, city managing director, and city council members prove their love and dedication to the people of this city and reallocate funds to directly benefit those in need.
+
+ Thank you for your time, 
+ [your name]
```
Comment (teddywilson) [inline _emails/philadelphia.md:38]: [NAME]

also make sure to include
[EMAIL]
[PHONE]
[ADDRESS]

**Comment by teddywilson** [comment]
Since we already have a philly letter with similar content, I'm going to close this one out.

Feel free to make a PR against the current philly template, or if you want to add a different one for philly that works to - e.g., defund12.org/philly-somethingelse. Check out Louisville as an example, as we have multiple letters for it!

Thanks so much!

**Pull request closed without merging**